Cargo.toml

```diff
@@ -15,6 +15,8 @@ Utilities for working with time-related functions in Rust.
 [dependencies]
 libc = "0.1"
 rustc-serialize = { version = "0.3", optional = true }
+postgres = { version = "*", optional = true }
```

Comment:
I believe that [postgres is responsible](https://github.com/sfackler/rust-postgres/blob/master/Cargo.toml#L36) for this functionality instead of time itself, so I don't think this should be added just yet.